go/mysql/server.go

```diff
@@ -320,23 +320,23 @@ func (l *Listener) handle(conn net.Conn, connectionID uint32, acceptTime time.Ti
 		case ComQuery:
 			queryStart := time.Now()
 			query := c.parseComQuery(data)
+			c.recycleReadPacket()
 			fieldSent := false
-			// sendFinished is set if a response has completed and
-			// no further packets must be sent.
-			sendFinished := false
+			// noStream is set if the response should just be an OK packet.
+			noStream := false
```

Comment:
'noStream' is even more confusing to me than 'sendFinished'. What we mean is we sent the final OK packet already, and no result was sent. How about:
sentOKPacket
responseHasNoRows
responseIsOKPacket
shortResponse
?

Reply:
I agree. I named it noStream when I was trying to refactor it differently, but that approach didn't work out. I think sendFinished is good enough.